test/e2e/scale.go

```diff
@@ -359,7 +359,7 @@ func ScaleSpec(ctx context.Context, inputGetter func() ScaleSpecInput) {
 				for i := range additionalClusterClassCount {
 					additionalName := fmt.Sprintf("%s-%d", input.ClusterClassName, i+1)
 					log.Logf("Apply additional ClusterClass %s/%s", namespace.Name, additionalName)
-					additionalClassYAML := bytes.Replace(clusterClassYAML, []byte(input.ClusterClassName), []byte(additionalName), -1)
+					additionalClassYAML := bytes.ReplaceAll(clusterClassYAML, []byte(scaleClusterNamePlaceholder), []byte(additionalName))
```

Comment:
Are we really sure that input.ClusterClassName should be changed to scaleClusterNamePlaceholder? I think it breaks the functionality